net/ip/mn_socket/include/mn_socket/mn_socket.h

```diff
@@ -187,6 +187,9 @@ int mn_close(struct mn_socket *);
 /*
  * Address conversion
  */
+
+#define MN_INET6_MAX_LEN    (39)    /* Does not include terminating \0. */
```

Comment:
To match the naming of the other items should use the following instead. Also note the size. And if adding this one, probably should add the MN_INET_ADDRSTRLEN there as well :)

#define MN_INET6_ADDRSTRLEN        46